esp/services/ws_fs/ws_fsService.cpp

```diff
@@ -1834,7 +1834,10 @@ bool CFileSprayEx::onSprayFixed(IEspContext &context, IEspSprayFixed &req, IEspS
 
         MemoryBuffer& srcxml = (MemoryBuffer&)req.getSrcxml();
         const char* srcip = req.getSourceIP();
-        const char* srcfile = req.getSourcePath();
+        StringBuffer sourcePath(req.getSourcePath());
```

Comment:
why don't we convert to posix after the test on line 1841-1847? No point in converting if those fields are empty and we're just going to throw an exception anyway.